changelogs/current.yaml

```diff
@@ -135,3 +135,7 @@ new_features:
     authentication failure status code and message in dynamic metadata.
 
 deprecated:
+- area: ext_authz
+  change: |
+    deprecated (1.2.5.0) :ref:`allowed_headers <envoy_v3_api_field_extensions.filters.http.ext_authz.v3.AuthorizationRequest.allowed_headers>` in favour
```

Comment:
Should this be 1.25.0?
Also, not sure if backporting is desired here or not, just wanted to see if it was considered.

Reply:
fixed the typo.

Reply:
yep, we are going to backport to the 1.25 patch release

/backport

Reply:
@rulex123 im wondering if we want to give a fuller ref to the deprecated/newer field - currently the changelog reads as

> ext_authz: deprecated (1.25.0) [allowed_headers](https://storage.googleapis.com/envoy-pr/a86daf7/docs/api-v3/extensions/filters/http/ext_authz/v3/ext_authz.proto.html#envoy-v3-api-field-extensions-filters-http-ext-authz-v3-authorizationrequest-allowed-headers) in favour of [allowed_headers](https://storage.googleapis.com/envoy-pr/a86daf7/docs/api-v3/extensions/filters/http/ext_authz/v3/ext_authz.proto.html#envoy-v3-api-field-extensions-filters-http-ext-authz-v3-extauthz-allowed-headers).

Reply:
which obviously makes sense if you click through the links etc - but would be good to show the  difference i think

Reply:
yep, makes sense @phlax